lib/graphql/upgrader/member.rb

```diff
@@ -39,16 +40,25 @@ def upgrade
             return_type.gsub! ',', ''
 
             transformable.sub!(field) do
-              f = "#{field_type == 'argument' ? 'argument' : 'field'} :#{name}, #{return_type}"
+              is_argument = ['argument', 'input_field'].include?(field_type)
+              f = "#{is_argument ? 'argument' : 'field'} :#{name}, #{return_type}"
 
               unless remainder.empty?
                 f += ',' + remainder
               end
 
-              if may_return_null
-                f += ', null: true'
+              if is_argument
+                if may_return_null
```

Comment:
Now that `may_return_null` also determines if an input is mandatory, maybe `has_bang` is a more appropriate name.

Reply:
Updated